__tests__/app.test.js

```diff
@@ -322,3 +322,28 @@ describe("PATCH:/api/articles/:article_id", () => {
             })
     })
 })
+describe("DELETE:/api/comments/:comment_id", ()=>{
+    test("204:should responds with status 204 no content  " ,()=>{
+        return request(app)
+        .delete('/api/comments/3')
+        .expect(204)
+        .then(()=>{
```

Comment:
Do we intend to do something here in this .then block? We could assert that we expect no body for example.